### PR TITLE
feat(frontend): Create the Discover Component (#120)

### PR DESCRIPTION
**Pull request by Rolly007**

### CODE REVIEW

File: frontend/src/views/Discover/Discover.module.scss [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+.title {
+  color: $thirdColor;
+  width: 40%;
+  padding-top: 40px;
+  margin-left: 10%;
+}
+
+.movies {
+  margin-top: 100px;
+}
```
Comment (revodark) [inline frontend/src/views/Discover/Discover.module.scss:53]: Please add a blank line

Comment (Rolly007) [reply frontend/src/views/Discover/Discover.module.scss:53]: thanks, I will add it

File: frontend/src/views/Discover/Discover.jsx [hunk 2]
```diff
@@ -0,0 +1,44 @@
+import React, { useState } from 'react';
+
+import Typography from '@material-ui/core/Typography';
+
+import CardsPagination from 'components/CardsPagination';
+import DiscoverForm from './DiscoverForm';
+
+import styles from './Discover.module.scss';
+
+const requestPath = '/movies/discover';
```
Comment (revodark) [inline frontend/src/views/Discover/Discover.jsx:10]: I think this should be in uppercase

File: frontend/src/views/Discover/DiscoverForm.jsx [hunk 3]
```diff
@@ -0,0 +1,207 @@
+/* eslint-disable camelcase */
+import React, { useMemo } from 'react';
+import { useSelector } from 'react-redux';
+import PropTypes from 'prop-types';
+
+import { MenuItem } from '@material-ui/core';
+import { TextField, Select, Autocomplete } from 'mui-rff';
+import get from 'lodash/fp/get';
+import set from 'lodash/set';
```
Comment (revodark) [inline frontend/src/views/Discover/DiscoverForm.jsx:9]: Maybe in this case we could do import {get, set} from 'lodash/fp, could you check it please?

File: frontend/src/components/NavBar/NavBar.jsx [hunk 4]
```diff
@@ -29,15 +32,50 @@ const NEW_LIST_TITLE = 'New List';
...
+      params: { list: 'languages' }
+    },
+    null,
+    AUTOMATIC
+  );
+
   const setSuccessSnackBar = useSnackBar(SUCCESS);
   const setErrorSnackBar = useSnackBar(ERROR);
 
+  useEffect(() => {
```
Comment (revodark) [inline frontend/src/components/NavBar/NavBar.jsx:73]: Hi, maybe we could perform this requests and dispatch in our routes index file, I think would be better, please let me know your comments

File: frontend/src/views/Discover/Discover.jsx (hunk 2 above)
Comment (Rolly007) [reply frontend/src/views/Discover/Discover.jsx:10]: ok, I will change it

File: frontend/src/components/NavBar/NavBar.jsx (hunk 4 above)
Comment (Rolly007) [reply frontend/src/components/NavBar/NavBar.jsx:73]: yes, you have reason, I will move to the "routes/indes.js" file, thanks

File: frontend/src/views/Discover/DiscoverForm.jsx (hunk 3 above)
Comment (Rolly007) [reply frontend/src/views/Discover/DiscoverForm.jsx:9]: I tried with **"set"** function of the **lodash/fp** but this function works different in comparision with **set** function of the **lodash library(without funcional programming)**, for this reason I decided to use the "set" function of lodash(without fp)

**Pull request merged**